src/client/lcd/api/TxAPI.ts

```diff
@@ -318,13 +317,16 @@ export class TxAPI extends BaseAPI {
       Broadcast.BLOCK
     ).then(d => {
       const blockResult: any = {
-        height: Number.parseInt(d.height),
         txhash: d.txhash,
         raw_log: d.raw_log,
         gas_wanted: Number.parseInt(d.gas_wanted),
         gas_used: Number.parseInt(d.gas_used),
       };
 
+      if (d.height) {
+        blockResult.height = Number.parseInt(d.height);
```

Comment:
The `Number` before `parseInt` is not need it. You can called like `parseInt(d.height, 10)`. Always add the radix  to it